azure-pipelines.yml

```diff
@@ -300,7 +300,7 @@ stages:
     # are the primary providers of the SciPy wheels available
     # on PyPI, so we now regularly test that the version of
     # _distributor_init.py in our wheels repo is capable of
-    # loading the DLLs from a master branch wheel build
+    # loading the DLLs from a main branch wheel build
```

Comment:
This refers to `scipy-wheels`?  Is that `master` or `main`?

Reply:
It looks like it is still master: https://github.com/MacPython/scipy-wheels